pkg/function/scale_workload.go

```diff
@@ -92,14 +93,15 @@ func (*scaleWorkloadFunc) Arguments() []string {
 		ScaleWorkloadNamespaceArg,
 		ScaleWorkloadNameArg,
 		ScaleWorkloadKindArg,
+		ScaleWorkloadWaitArg,
 	}
 }
 
-func getArgs(tp param.TemplateParams, args map[string]interface{}) (namespace, kind, name string, replicas int32, err error) {
+func getArgs(tp param.TemplateParams, args map[string]interface{}) (namespace, kind, name string, replicas int32, waitForReady bool, err error) {
 	var rep interface{}
 	err = Arg(args, ScaleWorkloadReplicas, &rep)
 	if err != nil {
-		return namespace, kind, name, replicas, err
+		return namespace, kind, name, replicas, waitForReady, err
```

Comment:
`waitForReady` is `false` here; probably wants to set it to `true` early. Similarly, at line 141.

Reply:
Done